### PR TITLE
Template update

### PR DESCRIPTION
**Pull request by sbilge**
Updated service template

### CODE REVIEW

**Comment by sbilge** [comment]
I cannot find why the Tests check fails to find the module "openpyxl". Any pointers? @lkuchenb

**Comment by lkuchenb** [comment]
You didn't run `./scripts/update_lock.py` after changing the `pyproject.toml` file.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "Intended Audience :: Developers",
+]
+dependencies = [
+    "typer >= 0.7.0",
+    "ghga-service-commons[api] >= 0.7.0",
+    "ghga-event-schemas >= 0.13.0",
+    "hexkit[akafka,s3,mongodb] >= 0.10.0",
+    "openpyxl==3.1.2",
+    "defusedxml==0.7.1",
+    "PyYAML==6.0"
```
Comment (lkuchenb) [inline pyproject.toml]: We don't have to pin these to exact versions anymore, pinning major should be fine.

Comment (lkuchenb) [inline pyproject.toml]:
```suggestion
    "openpyxl >= 3.1.2, == 3.*",
    "defusedxml ~= 0.7.1",
    "PyYAML == 6.*"
```

**Review by lkuchenb — CHANGES_REQUESTED** [review]

**Review by lkuchenb — APPROVED** [review]

**Pull request merged**